example-apps/internal-knowledge-search/app-ui/src/store/slices/searchApplicationSettingsSlice.ts

```diff
@@ -5,22 +5,40 @@ import {SearchApplicationSettingsModel} from "../../models/SearchApplicationSett
 
 const initialState: SearchApplicationSettingsModel = {
     appName: process.env.REACT_APP_SEARCH_APP_NAME || "some-search-application",
-    appUser: process.env.REACT_APP_SEARCH_USER || "elastic",
-    appPassword: process.env.REACT_APP_SEARCH_PASSWORD || "changeme",
+    indices: [],
     searchEndpoint: process.env.REACT_APP_SEARCH_APP_ENDPOINT || "https://some-search-end-point.co",
     searchPersona: "admin",
     searchPersonaAPIKey: "missing"
 };
 
-const searchApplicationSettingsSlice = createSlice({
+export const searchApplicationSettingsSlice = createSlice({
     name: 'searchApplicationSettings',
     initialState,
     reducers: {
+        updateAppName: (state, action: PayloadAction<string>) => {
```

Comment:
Ah, thanks for doing this. You'd left a comment for me to address this and I'd forgotten to.